src/Components/Web.JS/src/Platform/Mono/MonoPlatform.ts

```diff
@@ -176,39 +174,49 @@ function importDotnetJs(resourceLoader: WebAssemblyResourceLoader): Promise<Crea
     .keys(resourceLoader.bootConfig.resources.runtime)
     .filter(n => n.startsWith('dotnet.') && n.endsWith('.js'))[0];
   const dotnetJsContentHash = resourceLoader.bootConfig.resources.runtime[dotnetJsResourceName];
-  let src = `./${dotnetJsResourceName}`;
+  let src = new URL(`_framework/${dotnetJsResourceName}`, document.baseURI);
```

Comment:
Sorry, this is another breaking change. The `loadBootResource` callback needs to receive a string, otherwise existing code that does things like `value.endsWith(...)` will start failing.

I think we should continue to pass `_framework/blazor.webassembly.js` (not `./_framework/blazor.webassembly.js`) into `loadBootResource`, and then normalize the result against baseURI afterwards.

Reply:
I'm confused now, this is about `dotnet.*.js` not about `blazor.webassembly.js`. Also this new version doesn't pass `./` anymore. Also this is not about `resourceLoader`, we are not caching it at all, only browser does.
Could you please elaborate or propose a fix ?